pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -837,7 +837,7 @@ type StorageSpec struct {
 	// Defines the PVC spec to be used by the Prometheus StatefulSets.
 	// The easiest way to use a volume that cannot be automatically provisioned
 	// is to use a label selector alongside manually created PersistentVolumes.
-	VolumeClaimTemplate EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
+	VolumeClaimTemplates []EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
```

Comment:
Since Prometheus is a `v1` CRD, we can't remove a field without bumping the version (e.g. `v2`). In this case, we'd have to add a new field but not remove the previous one for compatibility purposes in order to stick to `v1`.

Reply:
Ok, get it!